script.js

```diff
@@ -1 +1,163 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+document.addEventListener('DOMContentLoaded', function () {
+  let countSchedule = 0;
+  let countDone = 0;
+
+  // current schedule&done
+  function makeHistoryElement(id) {
+    if (id == 'schedule') {
+      countSchedule += 1;
+      if (countDone > 0) {
+        countDone -= 1;
+      }
+    } else if (id == 'done') {
+      countDone += 1;
+      if (countSchedule > 0) {
+        countSchedule -= 1;
+      }
+    } else if (id == 'newSchedule') {
+      countSchedule += 1;
+    } else if (id == 'trashSchedule') {
+      countSchedule -= 1;
+    } else if (id == 'trashDone') {
+      countDone -= 1;
+    }
+
+    const historyH2 = document.createElement('h2');
+    historyH2.textContent = 'Current';
+
+    const historyDiv = document.createElement('div');
+    historyDiv.className = 'detail';
+
+    const schedule = document.createElement('i');
+    schedule.textContent = 'SCHEDULE';
+    const scheduleSpan = document.createElement('span');
+    scheduleSpan.textContent = countSchedule.toLocaleString();
+    scheduleSpan.className = 'countSchedule';
+    const done = document.createElement('i');
+    done.textContent = 'DONE';
+    const doneSpan = document.createElement('span');
+    doneSpan.textContent = countDone.toLocaleString();
+    doneSpan.className = 'countDone';
+
+    historyDiv.append(schedule, scheduleSpan, done, doneSpan);
+
+    const historyElement = document.querySelectorAll('.history')[0];
+    historyElement.innerHTML = '';
+    historyElement.append(historyH2, historyDiv);
+  }
+
+  // 오늘 날짜
+  function getTodayDate() {
+    const today = new Date();
+    const year = today.getFullYear();
+    const month = String(today.getMonth() + 1).padStart(2, '0');
+    const day = String(today.getDate()).padStart(2, '0');
+    const dateElement = document.querySelector('.date');
+    dateElement.textContent = `${year}.${month}.${day}`;
+  }
+
+  // 모달 띄우기, 닫기
+  const editModalBackground = document.querySelector('.edit-modal-bg');
+  const editBtn = document.querySelector('footer button');
+
+  editBtn.addEventListener('click', function () {
+    editModalBackground.style.display = 'flex';
+  });
+
+  function closeEditSchedule() {
+    document.getElementById('addDescription').value = '';
+    editModalBackground.style.display = 'none';
+  }
```

Comment:
모달 열고 닫는 로직을 하나의 함수로 통일하면 중복 코드를 제거 하고 함수 호출을 통해서 모달을 제어하는 방법도 가독성이랑 유지보수에 좋은 방법인 것 같아서 추천해봅니다!

```suggestion
function toggleModal(isOpen) {
  editModalBackground.style.display = isOpen ? 'flex' : 'none';
  if (!isOpen) document.getElementById('addDescription').value = '';
}

editBtn.addEventListener('click', () => toggleModal(true));
closeBtn.addEventListener('click', () => toggleModal(false));
```